benedict/dicts/io/io_dict.py

```diff
@@ -39,15 +39,17 @@ def _decode(s, format, **kwargs):
                 return data
             elif type_util.is_list(data):
                 # force list to dict
-                return {'values': data}
+                return {
```

Comment:
These changes are not related to this PR.

Reply:
This is probably because of my code formatter. I will be reverting it back.